### ISSUE TITLE
Support Request: Bump .NET runtime to 8.0 or later on Fedora 41

### ISSUE THREAD

**Issue by KelvinNovais** [OpenTabletDriver/OpenTabletDriver#3510]
### What do you need help with?

When I try to install the OTD with the official .rpm package, it returns:
`  - nothing provides dotnet-runtime-6.0 needed by opentabletdriver-0.6.4.0-1.x86_64 from @commandline`

Even if the [README](https://github.com/OpenTabletDriver/OpenTabletDriver?tab=readme-ov-file#all-platforms) metions .NET 8

### What have you tried?

Following the this [guide](https://learn.microsoft.com/en-us/dotnet/core/install/linux-fedora#install-net-6), it's still not possible to install the required package:
```
sudo dnf install dotnet-runtime-6.0
Updating and loading repositories:
Repositories loaded.
Failed to resolve the transaction:
No match for argument: dotnet-runtime-6.0
You can try to add to command line:
  --skip-unavailable to skip unavailable packages
```

Although the [docs](https://learn.microsoft.com/en-us/dotnet/core/install/linux-fedora#supported-distributions) doesn't mention Fedora 41, by running the following, only .NET 8.0 and 9.0 are supported:

```
dnf search -C dotnet-runtime
Updating and loading repositories:
Repositories loaded.
Matched fields: name
 dotnet-runtime-8.0.x86_64: NET 8.0 runtime
 dotnet-runtime-9.0.x86_64: NET 9.0 runtime
 dotnet-runtime-dbg-8.0.x86_64: Managed debug symbols NET 8.0 runtime
 dotnet-runtime-dbg-9.0.x86_64: Managed debug symbols NET 9.0 runtime

```

### Diagnostic Information

Not available.

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
Duplicate of #3485. See the milestone for what is left for a release. https://github.com/OpenTabletDriver/OpenTabletDriver/milestone/28.

You are welcome to build it yourself by cloning the repo, checking out 0.6.x and running the build script. Everything you need is within the README. Alternatively figure out how to get dotnet 6 on your version of fedora.

**Comment by gonX** [comment]
Fixed with #3368
